src/SurfaceFluxes.jl

```diff
@@ -557,7 +560,7 @@ function heat_exchange_coefficient(
     Δθ = TD.dry_pottemp(param_set, ts_in(sc)) - TD.dry_pottemp(param_set, ts_sfc(sc))
     ustar = compute_ustar(param_set, L_MO, sc, uft, scheme)
     θstar = Δθ * compute_physical_scale_coeff(param_set, sc, L_MO, UF.HeatTransport(), uft, scheme)
-    if Δθ ≈ FT(0) || windspeed(sc) ≈ FT(0)
+    if Δθ ≈ FT(0) || windspeed(sc) ≈ FT(0) || Δρ ≈ FT(0)
```

Comment:
Undefined variable?

Reply:
@yairchn, I've added
```julia
    ρ_in = TD.air_density(param_set, ts_in(sc))
    ρ_sfc = TD.air_density(param_set, ts_sfc(sc))
    Δρ = ρ_in - ρ_sfc
```
before this so that `Δρ` is defined. Can you confirm that this is correct?